Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -97,60 +130,69 @@ bool operator()(const woc::unique_cf<CFStringRef>& item1, const woc::unique_cf<C
+            ComPtr<IDWriteLocalizedStrings> displayName;
+            ComPtr<IDWriteLocalizedStrings> postScriptName;
+            BOOL exist;
 
-                if (SUCCEEDED(font->GetInformationalStrings(DWRITE_INFORMATIONAL_STRING_FULL_NAME, &displayName, &exist)) && exist) {
-                    info->displayName.reset(__CFStringCreateUppercaseCopy(_CFStringFromLocalizedString(displayName.Get()), locale.get()));
-                    woc::unique_cf<CFStringRef> uppercaseNameKey(__CFStringCreateUppercaseCopy(info->displayName.get(), locale.get()));
-                    ret.emplace(std::move(uppercaseNameKey), info);
-                }
+            if (SUCCEEDED(font->GetInformationalStrings(DWRITE_INFORMATIONAL_STRING_FULL_NAME, &displayName, &exist)) && exist) {
+                info->displayName.reset(__CFStringCreateUppercaseCopy(_CFStringFromLocalizedString(displayName.Get()), locale.get()));
+                woc::unique_cf<CFStringRef> uppercaseNameKey(__CFStringCreateUppercaseCopy(info->displayName.get(), locale.get()));
+                ret.emplace(std::move(uppercaseNameKey), info);
+            }
 
-                if (SUCCEEDED(font->GetInformationalStrings(DWRITE_INFORMATIONAL_STRING_POSTSCRIPT_NAME, &postScriptName, &exist)) &&
-                    exist) {
-                    info->postScriptName.reset(static_cast<CFStringRef>(CFRetain(_CFStringFromLocalizedString(postScriptName.Get()))));
-                    woc::unique_cf<CFStringRef> uppercaseNameKey(__CFStringCreateUppercaseCopy(info->postScriptName.get(), locale.get()));
-                    ret.emplace(std::move(uppercaseNameKey), info);
-                }
+            if (SUCCEEDED(font->GetInformationalStrings(DWRITE_INFORMATIONAL_STRING_POSTSCRIPT_NAME, &postScriptName, &exist)) && exist) {
+                info->postScriptName.reset(static_cast<CFStringRef>(CFRetain(_CFStringFromLocalizedString(postScriptName.Get()))));
+                woc::unique_cf<CFStringRef> uppercaseNameKey(__CFStringCreateUppercaseCopy(info->postScriptName.get(), locale.get()));
+                ret.emplace(std::move(uppercaseNameKey), info);
+            }
 
-                info->familyName.reset(CFStringCreateCopy(nullptr, familyName.get()));
+            info->familyName.reset(CFStringCreateCopy(nullptr, familyName));
 
-                info->weight = font->GetWeight();
-                info->stretch = font->GetStretch();
-                info->style = font->GetStyle();
-            }
+            info->weight = font->GetWeight();
+            info->stretch = font->GetStretch();
+            info->style = font->GetStyle();
         }
+    }
 
-        return ret;
-    }();
+    return ret;
+}
+
+/**
+ * Private static map, that maps display name and postscript name to a _DWriteFontProperties struct for the corresponding user defined font.
+ * Kept separate from c_systemFontPropertiesMap because this needs to be edited whenever a font is (un)registered
```

Comment:
Since it's not used anywhere else I'd like to keep it function local, and as @DHowett-MSFT and @rajsesh-msft learned me we run into fun DLL/Com loading stuff if it's file static